### PR TITLE
Add interactive Quickstart for iOS and macOS [SDK-3299]

### PR DESCRIPTION
**Pull request by Widcket**
### Changes

This PR migrates the Auth0.swift Quickstart to the new interactive format. The old quickstart was updated and slightly tweaked as well.

Screen Shot 2022-05-12 at 02 06 40

Screen Shot 2022-05-12 at 02 07 27

Screen Shot 2022-05-12 at 02 07 50

### CODE REVIEW

File: articles/quickstart/native/ios-swift/interactive.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+To use this quickstart, you’ll need to:
+* Sign up for a free Auth0 account or log in to Auth0.
+* Have an existing iOS / macOS app that you want to integrate with. Alternatively, you can view or download a sample app after logging in.
+
+## Configure Auth0 {{{ data-action=configure }}}
+
+To use Auth0 services, you’ll need to have an application set up in the Auth0 Dashboard. The Auth0 application is where you will configure how you want authentication to work for the app you are developing.
+
+### Configure an Auth0 Application
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide (word list): titles should be sentence case and "application" should not be capitalized

```suggestion
### Configure an Auth0 application
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+Use the interactive selector to create a new Auth0 application or select an existing application that represents the iOS or macOS app you want to integrate with. Every application in Auth0 is assigned an alphanumeric, unique Client ID that your app will use to call Auth0 APIs through the SDK.
+
+Any settings you configure using this quickstart will automatically update for your Auth0 application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your applications in the future.
+
+If you would rather explore a complete configuration, you can view a sample app instead.
+
+### Configure Callback and Logour URLs
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide: titles should be sentence case (also a typo):

```suggestion
### Configure callback and logout URLs
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+YOUR_BUNDLE_IDENTIFIER://${account.namespace}/macos/YOUR_BUNDLE_IDENTIFIER/callback
+```
+
+E.g. if your iOS bundle identifier was `com.company.myapp` and your Auth0 tenant's domain was `company.us.auth0.com`, then this value would be:
+
+```text
+com.company.myapp://company.us.auth0.com/ios/com.company.myapp/callback
+```
+
+## Configure Your App
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide: titles should be sentence case

```suggestion
## Configure your app
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+Sorry about that. Here's a couple things to double check:
+- Make sure the correct Auth0 application is selected.
+- Did you press the **Save** button after entering your URLs?
+- Make sure the Auth0 Domain and Client ID are set correctly.
+
+Still having issues? Check out our [documentation]() or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
+
+## Add Login to Your App {{{ data-action=code data-code="ContentView.swift#21:33" }}}
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide: titles should be sentence case

```suggestion
## Add login to your app {{{ data-action=code data-code="ContentView.swift#21:33" }}}
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+:::checkpoint-failure
+Sorry about that. Here's a couple things to double check:
+- Did you enter the correct callback URL value?
+- Make sure the Auth0 Domain and Client ID are set correctly.
+
+Still having issues? Check out our [documentation]() or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
+
+## Add Logout to Your App {{{ data-action=code data-code="ContentView.swift#35:46" }}}
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide: titles should be sentence case

```suggestion
## Add logout to your app {{{ data-action=code data-code="ContentView.swift#35:46" }}}
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+:::checkpoint-failure
+Sorry about that. Here's a couple things to double check:
+- Did you enter the correct logout URL value?
+- Make sure the Auth0 Domain and Client ID are set correctly.
+
+Still having issues? Check out our [documentation]() or visit our [community page](https://community.auth0.com) to get more help.
+:::
+::::
+
+## Access User Profile Information {{{ data-action=code data-code="User.swift#15:18" }}}
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide: titles should be sentence case

```suggestion
## Access user profile information {{{ data-action=code data-code="User.swift#15:18" }}}
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+The `Credentials` instance you obtained after logging in includes an [ID Token](/secure/tokens/id-tokens). The ID Token contains the profile information associated with the logged-in user, such as their name or profile picture. You can use these details to personalize the user interface of your app.
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide (word list): ID token should be lowercase

```suggestion
The `Credentials` instance you obtained after logging in includes an [ID token](/secure/tokens/id-tokens). The ID token contains the profile information associated with the logged-in user, such as their name or profile picture. You can use these details to personalize the user interface of your app.
```

File: articles/quickstart/native/ios-swift/interactive.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+
+The Auth0.swift SDK includes a [utility](https://github.com/auth0/JWTDecode.swift) for decoding [JWTs](https://jwt.io/) like the ID Token. Start by importing the `JWTDecode` module in the file where you want to access the user profile information. Then, use the `decode(jwt:)` method to decode the ID Token and access the claims it contains.
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/interactive.md]: Style guide (word list): ID token should be lowercase

```suggestion
The Auth0.swift SDK includes a [utility](https://github.com/auth0/JWTDecode.swift) for decoding [JWTs](https://jwt.io/) like the ID token. Start by importing the `JWTDecode` module in the file where you want to access the user profile information. Then, use the `decode(jwt:)` method to decode the ID token and access the claims it contains.
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
LGTM, just a few suggestions to align with the style guide and word list

**Comment by Widcket** [comment]
Perfect, thank you!

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**